localSetup/projects/nginx/nginx-conf.d/contentauthor.conf

```diff
@@ -21,10 +21,10 @@ server {
 	server_name contentauthor.local;
 
 	location / {
-		proxy_pass http://contentauthor:8081;
+		set $upstream http://contentauthor:8081;
+		proxy_pass $upstream;
 		proxy_set_header X-Forwarded-For  $remote_addr;
 		proxy_set_header Host $host;
-		proxy_redirect default;
```

Comment:
Not compatible with a dynamic `proxy_pass`, hopefully this doesn't break anything...